.github/workflows/ci.yaml

```diff
@@ -25,4 +25,13 @@ jobs:
       - name: Run checks
         env:
           FORCE_COLOR: 1
-        run: earthly --ci +checks
\ No newline at end of file
+        run: earthly --ci +checks
+
+  docker-build:
+    name: Build Docker Image
+    timeout-minutes: 10
+    runs-on: [self-hosted, linux, x64]
```

Comment:
I _think_ there was a reason that this was not self-hosted as it was a public repo and could potentially lead to extra costs if abused. Now how valid that is I am not sure but that was my vague recollection of this being the case before. Just FYI